src/components/Header/Navigation.js

```diff
@@ -115,18 +115,17 @@ function Navigation({ classes, width, dominion }) {
   const { open: openContact, toggleModal: toggleContact } = useToggleModal(
     'contact'
   );
-  const renderMenuList = () => (
+  const navList = [
+    { title: 'About', link: '/about' },
+    { title: 'Showcase', link: `#showcase` },
+    { title: 'Resources', link: '/resources' },
+    { title: 'Contact', onClick: toggleContact }
+  ]; // --> off
+
+  /* eslint react/no-array-index-key: 0 */ const renderMenuList = () => (
```

Comment:
Not sure what's going on here... I've read the documentation and looks like we should be using keys?

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

https://reactjs.org/docs/lists-and-keys.html#keys